### PR TITLE
Example cnf

### PR DESCRIPTION
**Pull request by isaacdorfman**

### CODE REVIEW

**Comment by isaacdorfman** [comment]
@jun-rh

File: example-cnf/cr/traffic.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+// ReelFirst returns a reel.Step which expects output from running the Args command.
+func (r *Traffic) ReelFirst() *reel.Step {
+	return &reel.Step{
+		Expect:  []string{trafficPassedSuccessfullyRegex},
+		Timeout: r.timeout,
+	}
+}
+
+// ReelMatch determines whether the container is based on Red Hat technologies through pattern matching logic.
+func (r *Traffic) ReelMatch(_, _, _ string) *reel.Step {
```
Comment (HamadiSE) [inline example-cnf/cr/traffic.go:80]: if you're not storing any data and only returning success/failure, it's better to write the test in JSON.

File: example-cnf/cr/create.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+// ReelFirst returns a reel.Step which expects output from running the Args command.
+func (r *Create) ReelFirst() *reel.Step {
+	return &reel.Step{
+		Expect:  []string{createdRegex},
+		Timeout: r.timeout,
+	}
+}
+
+// ReelMatch determines whether the container is based on Red Hat technologies through pattern matching logic.
+func (r *Create) ReelMatch(_, _, _ string) *reel.Step {
```
Comment (HamadiSE) [inline example-cnf/cr/create.go:80]: the logic in this block seems easy and can be handled with JSON.
Any reason why you're using new handler ?

File: example-cnf/suite.go [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+				gomega.Expect(err).To(gomega.BeNil())
+				gomega.Expect(test).ToNot(gomega.BeNil())
+				result, err := test.Run()
+				gomega.Expect(err).To(gomega.BeNil())
+				gomega.Expect(result).To(gomega.Equal(tnf.SUCCESS))
+
+				const sleepTime = 40
+				time.Sleep(time.Second * sleepTime)
+
+				trafficPassed := cr.NewTraffic(testCustomResourceNamespace, testCustomResourceCRType, testCustomResourceCRName, testTimeout)
```
Comment (HamadiSE) [inline example-cnf/suite.go:97]: identifiers for both tests are missing

**Comment by pkliczewski** [comment]
@isaacdorfman Please rebase

**Pull request closed without merging**